.pre-commit-config.yaml

```diff
@@ -174,3 +174,7 @@ repos:
     -   id: codespell
         types_or: [python, rst, markdown]
         files: ^pandas/core/
+-   repo: https://github.com/asottile/setup-cfg-fmt
+    rev: v1.16.0
+    hooks:
+    -   id: setup-cfg-fmt
```

Comment:
can you do this as a precursor

Reply:
I tried it. But this hook requires a [metadata] section.